lib/inventory/metadata/get.go

```diff
@@ -44,7 +44,7 @@ func Get(ctx context.Context) (*Metadata, error) {
 		go func() {
 			defaultFetcher := &fetchConfig{}
 			defaultFetcher.setDefaults()
-			metadata = defaultFetcher.fetch()
+			metadata = defaultFetcher.fetch(ctx)
```

Comment:
If we do it like this, we end up never having metadata if the very first call to `Get` is interrupted.

Reply:
I don't think this is an issue in practice. The context used is the control stream's close context.